packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/upgrade-version/upgrade-version.controller.js

```diff
@@ -12,6 +12,10 @@ export default class {
   $onInit() {
     this.upgradingVersion = false;
     this.selectedVersion = null;
+    this.trackDatabases(
+      'dashboard::general_information::upgrade_version',
```

Comment:
This prefix will be different on the next lot and won't be the same all the time anymore. Is it ok to let it like that knowing that it will be done differently in the following PR ?

Reply:
sure plz tale care of it next PR if the prefix is same.